uwuno/src/main/java/com/learning/uwuno/room.java

```diff
@@ -1,16 +1,17 @@
 package com.learning.uwuno;
 
 import java.util.ArrayList;
+import java.util.UUID;
 
 public class room {
     // Class Variables
-    final private int uid;
+    final private String uid;
     private String roomName;
     private ArrayList<player> playerList = new ArrayList<player>();
 
     // Class functions
-    public room(int id, String roomName) {
-        this.uid = id;
+    public room(String roomName) {
```

Comment:
Did you ever find a way to shorten the id returned.

Reply:
Nope. I put it as a task on trello.